### PR TITLE
Add server restart/stop to quick bar command list

### PR DESCRIPTION
**Pull request by donkawechico**
## Proposed change

Separating the server control commands out of https://github.com/home-assistant/frontend/pull/7380 since navigation commands became a bit more complex.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/quick-bar/ha-quick-bar.ts [hunk 1]
```diff
@@ -297,6 +302,41 @@ export class QuickBar extends LitElement {
...
+      this._generateConfirmationCommand(
+        {
+          text: this.hass.localize(
+            "ui.dialogs.quick-bar.commands.server_control.perform_action",
+            "action",
+            this.hass.localize(
+              `ui.dialogs.quick-bar.commands.server_control.${action}`
+            )
+          ),
+          icon: "hass:robot",
```
Comment (bramkragten) [inline src/dialogs/quick-bar/ha-quick-bar.ts:318]: `hass:robot` is used for automations

Comment (donkawechico) [reply src/dialogs/quick-bar/ha-quick-bar.ts:318]: What do we use for server stuff

Comment (bramkragten) [reply src/dialogs/quick-bar/ha-quick-bar.ts:318]: `mdi:stop` and `mdi:restart`?

Comment (donkawechico) [reply src/dialogs/quick-bar/ha-quick-bar.ts]: done

**Review by bramkragten — APPROVED** [review]

**Pull request merged**